docs/topics/configmap-mission-basic-interaction-spring-boot-tomcat.adoc

```diff
@@ -38,33 +38,10 @@ NAME                             READY     STATUS      RESTARTS   AGE
 +
 You `{app-name}-1-aaaaa` pod should have a status of `Running` once it's fully deployed and started.
 
-. Use `curl` to execute a `GET` request against the booster with the updated ConfigMap configuration. You can also use a browser to do this.
+. Use `curl` to execute a `GET` request against the booster with the updated ConfigMap configuration. You can also use a browser to do this. You should see your updated greeting.
```

Comment:
_should_ is discouraged because it's a loaded word. I recommend rewording to _Your updated greeting is shown._

If we want to hint at a possibility of a failure, we should offer some troubleshooting IMO.

Applies wherever this sentence was added.

Reply:
This is section is rather old, and I was unsure whether to update it in this PR. But I see your point, and personally, I don't really like the wording here, either. Will update. in the next pass